host/linux/windows.c

```diff
@@ -20,15 +20,15 @@ HMODULE LoadLibraryExA(LPCSTR path, HANDLE file, DWORD flags)
         goto done;
 
     /* Allocate memory to hold the image (align on a page boundary). */
-    if (!(data = memalign(OE_PAGE_SIZE, st.st_size)))
+    if (!(data = memalign(OE_PAGE_SIZE, (size_t)st.st_size)))
```

Comment:
I guess I don't understand what this file is... linux/windows.c sounds like a contradiction